packages/e2e-tests/specs/site-editor/style-variations.test.js

```diff
@@ -22,19 +22,22 @@ async function getAvailableStyleVariations() {
 	return page.$$( VARIATION_ITEMS_STYLES_SELECTOR );
 }
 
-async function applyVariation( number ) {
+async function applyVariation( title ) {
 	await toggleGlobalStyles();
 	await openOtherStyles();
-	const variations = await getAvailableStyleVariations();
-	await variations[ number ].click();
+	const variation = await await page.waitForXPath(
+		`//*[@role="button"][@aria-label="${ title }"]`
+	);
+	await variation.focus();
+	await page.keyboard.press( 'Enter' );
```

Comment:
Why focus and press instead of click?

Reply:
Because of the height issue https://github.com/WordPress/gutenberg/pull/39737.

* I will revert this to click now that it's resolved.
* Update PR description.